astronomer/providers/amazon/aws/example_dags/example_redshift_cluster_management.py

```diff
@@ -24,6 +25,18 @@
 ) as dag:
     start = DummyOperator(task_id="start")
 
+    # Execute AWS command then sleep for 5 min so that cluster would be available
+    create_redshift_cluster = BashOperator(
+        task_id="create_redshift_cluster",
+        bash_command="aws redshift create-cluster "
+        "--db-name dev "
+        "--cluster-identifier redshift-cluster-1 "
+        "--cluster-type single-node "
+        "--node-type dc2.large  "
+        "--master-username adminuser "
+        "--master-user-password TopSecret1 && sleep 5m",
```

Comment:
The password shouldnt be checked in, and could be managed through environment variables if possible.

Reply:
Fixed. Now, it is coming from env